galaxy-deps/values.yaml

```diff
@@ -1,7 +1,13 @@
-postgresql:
```

Comment:
This change will affect the README in this repo as well as this conditional: https://github.com/galaxyproject/galaxy-helm-deps/blob/2b81840d3e8375ba0cc3ddee27bf74735026ca97/galaxy-deps/templates/cnpg-wait.yml#L1

And consequently https://github.com/galaxyproject/galaxykubeman-helm/pull/89/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R15